rsocket-core/src/test/java/io/rsocket/loadbalance/LoadbalanceTest.java

```diff
@@ -26,10 +29,13 @@
 
 public class LoadbalanceTest {
 
-  @Test
-  public void shouldDeliverAllTheRequestsWithRoundRobinStrategy() {
+  @BeforeEach
+  void setUp() {
     Hooks.onErrorDropped((__) -> {});
```

Comment:
I guess I forgot to do `Hooks.resetOnErrorDropped()` can you please add that one in the `@AfterEach`